modules/get-started/pages/release-notes/redpanda.adoc

```diff
@@ -20,6 +20,10 @@ Redpanda now supports the following Kafka APIs for managing SASL user credential
 
 See also: xref:manage:security/authentication.adoc#sasl[Configure Authentication] and xref:manage:security/authorization/acl.adoc[]
 
+== Schema Registry Authorization
+
+You can now use the Enterprise-licensed feature xref:manage:schema-reg-authorization.adoc[Schema Registry Authorization] to control access to Schema Registry subjects and operations using either `rpk` or the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API] endpoints. Schema Registry Authorization offers more granular control over who can do what with your Redpanda Schema Registry data. ACLs used for Schema Registry access also support RBAC roles.
```

Comment:
@Feediver1 'resources' is a better term than 'data' here (like you used above in the licensing section).  'data' confuses this a bit with messages in a topic